types/index.d.ts

```diff
@@ -1,21 +1,84 @@
+/**
+ * A database transaction
+ */
 export type Transaction = Array<object>;
+/**
+ * The Schema is a way to to simplify relational queries at query time.
+ * Presently, it allows you to define relationships and uniqueness constraints.
+ */
 export type Schema = object;
+/**
+ * Homebase configuration.
```

Comment:
It would also be nice to have some examples of what the data should look like https://jsdoc.app/tags-example.html